tests/all/integration/api/event/test_event_view_access.py

```diff
@@ -59,3 +61,16 @@ def test_event_draft_get_admin(db, client, admin_jwt):
     )
 
     assert response.status_code == 200
+
+
+def test_event_get_user_role(client, db, user, jwt):
+    event = get_event(db, user)
+
+    response = client.get(
+        f'/v1/events/{event.id}?include=roles.user',
+        content_type='application/vnd.api+json',
+        headers=jwt,
+    )
+
+    assert response.status_code == 200
+    assert json.loads(response.data)['included'][1]['type'] == 'user'
```

Comment:
Please check user ID and name of role is owner

Reply:
Ohh okay sure! :v:

Reply:
@iamareebjamal for querying the roles.user and roles.role we have to make two separate calls then since `?include=roles.role,roles.user` only fetches for one. Should I add a separate call then or just check for User id here?

Reply:
Ok, then let me think